tests/components/application-connector/resources/charts/gateway-test/templates/applications/credentials/mtls-oauth-nagative-other-ca.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mtls-oauth-negative-other-ca
+  namespace: kyma-integration
+type: Opaque
+data:
+  {{- $files := .Files }}
+  crt: {{ $files.Get "certs/negative/client.crt" | b64enc  }}
+  key: {{ $files.Get "certs/negative/client.key" | b64enc  }}
+  clientId: {{ "someClientID" | b64enc }}
```

Comment:
I'll log an issue for the Application Gateway cache problem if there is `clientId: {{ "ClientID" | b64enc }}`. I write it just to remind myself about it :D